oxo.yaml

```diff
@@ -1,34 +1,34 @@
 kind: Agent
-name: semgrep
+name: opengrep
 version: 1.0.4
 description: |
-  This repository is an implementation of [OXO Agent](https://pypi.org/project/ostorlab/) for [Semgrep](https://github.com/returntocorp/semgrep) by r2c.
+  This repository is an implementation of [OXO Agent](https://pypi.org/project/ostorlab/) for [Opengrep](https://github.com/opengrep/opengrep) by r2c.
   ## Getting Started
   To perform your first scan, simply run the following command:
   ```shell
-  oxo scan run --install --agent agent/ostorlab/semgrep file code.c
+  oxo scan run --install --agent agent/ostorlab/opengrep file code.c
   ```
-  This command will download and install `agent/ostorlab/semgrep` and analyze the file `code.c`.
+  This command will download and install `agent/ostorlab/opengrep` and analyze the file `code.c`.
   For more information, please refer to the [OXO Documentation](https://oxo.ostorlab.co/docs)
   ## Usage
-  Agent Semgrep can be installed directly from the oxo agent store or built from this repository.
+  Agent Opengrep can be installed directly from the oxo agent store or built from this repository.
 
    ### Install directly from oxo agent store
    ```shell
-   oxo agent install agent/ostorlab/semgrep
+   oxo agent install agent/ostorlab/opengrep
    ```
   You can then run the agent with the following command:
   ```shell
-  oxo scan run --install --agent agent/ostorlab/semgrep file code.c
+  oxo scan run --install --agent agent/ostorlab/opengrep file code.c
   ```
   ### Build directly from the repository
-   1. To build the semgrep agent you need to have [oxo](https://pypi.org/project/ostorlab/) installed in your machine.  if you have already installed oxo, you can skip this step.
+   1. To build the Opengrep agent you need to have [oxo](https://pypi.org/project/ostorlab/) installed in your machine.  if you have already installed oxo, you can skip this step.
   ```shell
   pip3 install ostorlab
   ```
    2. Clone this repository.
   ```shell
-  git clone https://github.com/Ostorlab/agent_semgrep.git && cd agent_semgrep
+  git clone https://github.com/Ostorlab/agent_opengrep.git && cd agent_opengrep
```

Comment:
Keep the same repo for the moment. All the breaking changes should be done after we see the agent working.